README.md

```diff
@@ -1,39 +1,82 @@
-<div align="center"><h1>Propel UI Kit</h1>
-<h2>Building Better Data Visualizations with Ease.</h2>
+<br />
+<div align="center">
+<h1>Propel UI Kit</h1>
+    <h2>React components for dashboards and data visualizations</h2>
+  <img src="public/images/UI-kit-header.png"/>
+  <h3 align="center">
+    <a href="https://propelify.propeldata.com" target="_blank">Demo Dashboard</a> |&nbsp; 
+    <a href="https://storybook.propeldata.com" target="_blank">Storybook</a>
+  </h3>
 </div>
 <br />
 
-## Introduction
+The Propel UI Kit is an open-source library of React components to quickly build awesome-looking dashboards and data visualizations.
+
+## Why the Propel UI Kit?
+
+Building data visualization UIs is never easy. Implementing designs from scratch is time-consuming, fetching data requires a lot of backend logic, and there are always a lot of edge cases to deal with, like the intricacies of the different time ranges, or handling the loading and error states.
+
+Charting libraries like ChartJS, Echarts, Highcharts, or D3 are great but still need a lot of work to build them into elegant and cohesive product experiences. On the flip side, embedding third-party chart widgets via iFrames into your customer-facing product results in an inflexible, slow, and inelegant product experience.
+
+That's why we built the Propel UI Kit. Its component-based approach allows you to move fast, look great, and focus your efforts on building components that are unique to your product.
+
+## Core principles
```

Comment:
In summary, what I see under the next two sections is

**Core principles**

- Don't reinvent the wheel.
- Sensible defaults.
- Customizable.
- For React.
- Propel is optional.

**Key features**

- Responsive.
- Handles error and loading states.
- Works with Propel (i.e., fetches the data).

---

IMO, some of these could be combined, distilled, and/or better distinguished. Maybe we only need "Key features". Most important to me feel like:

- Don't reinvent the wheel.
- Sensible defaults, but customizable.
- Propel is optional.

React is implied. Responsiveness, accessibility, plus error and loading states are sort of table stakes, IMO. Especially error and loading states, which are already alluded to.

Reply:
Distinguishing what the Propel UI Kit does vs. a plain charting library is key to land right away. The features section aims to make this clear and I would not assume it is table stakes as these features are what distinguish the UI Kit from a charting library